examples/simple_trainer.py

```diff
@@ -55,6 +55,8 @@ class Config:
     global_scale: float = 1.0
     # Normalize the world space
     normalize_world_space: bool = True
+    # Camera model
+    camera_model: str = "pinhole"
```

Comment:
minor note: it would be nice to use `Literal["fisheye", "pinhole"]` here and elsewhere!